Bump virgo server version to 3.7.2

zoliMolnar opened the pull request.

Commit:
[improvement] Bump virgo server version to 3.7.2, change config files...

... to match the new version

cmark commented on releng/com.b2international.snowowl.server.update/assembly/common/bin/dmk.sh:
```diff
@@ -186,52 +175,40 @@ then
+		mkdir -p "$TMP_DIR"
 		
-		#Added awt.headless - http://mail-archives.apache.org/mod_mbox/poi-user/200705.mbox/%3C15719338671.20070504144714@dinom.ru%3E
-        JAVA_OPTS="$JAVA_OPTS \
```
This was our custom default JVM configuration, please bring it back. Also check the other files as well, again!

Commit:
[improvement] Change suffix in tomcat-server.xml add javax.servlet...

... package imports to org.eclipse.virgo.kernel.userregion.properties
added java options to dmk.sh

apeteri commented on releng/com.b2international.snowowl.server.update/assembly/common/configuration/tomcat-server.xml:
```diff
@@ -15,43 +15,62 @@
-               connectionTimeout="20000"
-			   asyncTimeout="60000"
-               redirectPort="8443" />
+    <Connector address="127.0.0.1" port="8080" 
```
This will make the non-HTTPS port listen on the local interface only; please change the `address` value to `0.0.0.0`.

Commit:
[improvement] Change address in tomcat-server.xml

apeteri reviewed (approved):
👍

zoliMolnar commented:
Thanks

cmark reviewed (approved):
LGTM!

nagyo reviewed (approved):
👍
